src/braket/circuits/gates.py

```diff
@@ -1627,7 +1787,15 @@ def cphaseshift00(
 
 
 class CPhaseShift01(AngledGate):
-    """Controlled phase shift gate for phasing the \\|01> state.
+    r"""Controlled phase shift gate for phasing the \|01> state.
+
+    Unitary matrix:
+        .. math:: \mathtt{CPhaseShift01}(\phi) = \begin{bmatrix}
+                e^{i \phi} & 0 & 0 & 0 \\
+                0 & 1 & 0 & 0 \\
```

Comment:
should the `e^{i \phi}` be in the second row here?
```suggestion
                1 & 0 & 0 & 0 \\
                0 & e^{i \phi} & 0 & 0 \\
```